scripts/update/radarr.sh

```diff
@@ -11,19 +11,19 @@ if [[ -f /install/.radarr.lock ]]; then
         [[ -z $radarrOwner ]] && radarrOwner=$(_get_master_username)
 
         if [[ $(_radarr_version) = "mono-v3" ]]; then
-            echo_progress_start "Downloading source files"
+            echo_progress_start "Downloading release files"
             if ! curl "https://radarr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore&arch=x64" -L -o /tmp/Radarr.tar.gz >> "$log" 2>&1; then
                 echo_error "Download failed, exiting"
                 exit 1
             fi
-            echo_progress_done "Source downloaded"
+            echo_progress_done "Release downloaded"
 
             isactive=$(systemctl is-active radarr)
             echo_log_only "Radarr was $isactive"
             [[ $isactive == "active" ]] && systemctl stop radarr -q
 
-            echo_progress_start "Extracting archive"
-            rm /opt/Radarr/Radarr.exe
+            echo_progress_start "Extracting archive and removing old binaries"
```

Comment:
If we're nitpicking these echoes, "removing old binaries" is not at all what is being done here.

Reply:
This is exactly what is being done here; no action needed

we delete the old binaries in /opt/Radarr  and then extract the new tarball.

In fact these are from the official .net upgrade instructions
>Delete your existing binaries (contents or folder of /opt/Radarr) and replace with the contents of the .tar.gz you just downloaded.
>sudo mv /opt/Radarr /opt/Radarr.old Backup the old Binaries
>sudo rm -rf /opt/Radarr.old Remove the old binaries
https://wiki.servarr.com/Radarr_System#update_to_net_core_version

Reply:
`/opt/Radarr` is a directory, not a binary ;)

Plus, if we're doing this recursively, it would make sense to split them up in case the removal or the extrction take longer amounts of time to complete.

Reply:
the directory contains the radarr binaries and hence that is what the development team refers to this as "removing the radarr binaries"

semantics aside, so what are you looking for here?  the whole folder must be removed and replaced.